python/docs/source/getting_started/ps_install.rst

```diff
@@ -1,145 +0,0 @@
-============
```

Comment:
I think we can keep this and ps_videos_blogs.rst for now. They will be either updated, removed or merged at SPARK-35645

Reply:
Okay, restored.